Packs/RecordedFutureV3/Integrations/RecordedFutureAlerts/README.md

```diff
@@ -1,7 +1,272 @@
+XSOAR.  
+The integration allows you to:
 
+* Search and fetch alerts from the Recorded Future platform.
+* Update alert status, assignee and comment/note from inside XSOAR.
+* Automatically fetch screenshots that accompany the alert.
 
-You auto-generate this README file from your integration YML file using the `demisto-sdk generate-docs` command.
+## Configure Recorded Future Alerts in Cortex
+
+| **Parameter**                         | **Description**                                                                                                                                                          | **Required** |
+|---------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------|
+| Fetch incidents                       |                                                                                                                                                                          | False        |
+| Incident type                         |                                                                                                                                                                          | False        |
+| Your server URL                       |                                                                                                                                                                          | True         |
+| API Key                               | The API Key to use for the connection                                                                                                                                    | True         |
+| Source Reliability                    | Reliability of the source providing the intelligence data.                                                                                                               | False        |
+| Incidents fetch interval              |                                                                                                                                                                          | False        |
+| Maximum number of incidents per fetch | The max number of incidents to fetch per run. Cannot be bigger than 50.                                                                                                  | False        |
+| First fetch time                      | How far back we should fetch alerts on the first incident fetch. This value is not used for subsequent fetches. Cannot be bigger than 90 days.                           | False        |
+| Enable Classic Alerts                 |                                                                                                                                                                          | False        |
```

Comment:
Can we edit this file? Or is it generated and will be overwritten?

Would be nice to describe e.g. this param in the readme, but not show tooltip in the integration